Makefile

```diff
@@ -1,10 +0,0 @@
-SHELL := /bin/bash
-
-# List of targets the `readme` target should call before generating the readme
-export README_DEPS ?= docs/targets.md docs/terraform.md
-
--include $(shell curl -sSL -o .build-harness "https://git.io/build-harness"; echo .build-harness)
-
-## Lint terraform code
-lint:
-	$(SELF) terraform/install terraform/get-modules terraform/get-plugins terraform/lint terraform/validate
```

Comment:
@atanas-lazarov Do we have to remove all of these files? Ideally, we should minimize the differences between our fork and the upstream repo for two reasons:

1. It makes it clear/transparent what has changed by reducing the amount of noise in the diffs
2. We're less likely to run into conflicts if we rebase onto the upstream to integrate bug/security fixes, etc.